README.md

```diff
@@ -68,18 +67,29 @@ Note: This is an example of a non-normative note.
 
 # Overview
 
-Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP. Carrying GraphQL in HTTP does not mean that GraphQL overrides existing
-semantics of HTTP but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
+Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP.
+Carrying GraphQL in HTTP does not mean that GraphQL overrides existing semantics of HTTP,
+but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
 
-GraphQL naturally follows the HTTP request/response message model providing a GraphQL request in an HTTP request and
-GraphQL response in an HTTP response.
+GraphQL naturally follows the HTTP request/response message model,
+providing a GraphQL request in an HTTP request and GraphQL response in an HTTP response.
 
 # URL
 
-A GraphQL server operates on a single URL and all GraphQL requests for a given service should be directed
-at this URL. Other protocols may also use that URL.
+A GraphQL over HTTP compliant server MUST designate at least one URL that handles GraphQL requests.
+
+It is allowed for the same GraphQL schema to be available on multiple URLs.
```

Comment:
I wonder if this line is needed (I doubt that many people have a need to run the same schema more than once on a server).  It might help to be more generic.  Something that tells people that each URL that handles GraphQL requests MUST be an independent GraphQL endpoint (even one with the same schema), as long as they are each complete.

How about:

```
Multiple URLs MAY exist on a server to handle GraphQL requests.  These URLs MUST be independent and complete.
```

Reply:
I think this is valuable to avoid ambiguity in scenarios where a server migrates its GraphQL endpoint to a different URL, offering both in a transitional period.

Reply:
Do you think the sentence above covers the completeness aspect?

> All GraphQL operations that are available in the schema MUST be available on the URLs dedicated to it.

Reply:
Added the first part of your suggestion.

I'd rather have the spec be a bit more wordy and possibly redundant than to be ambiguous.